user/languages/javascript-with-nodejs.md

```diff
@@ -121,6 +121,51 @@ npm install
 
 Note that `npm install` can fail if a shrinkwrapped git dependency pointing to a branch has its HEAD changed.
 
+## Ember Apps
+You can build your Ember Apps on Travis CI and can automate test. By default ember comes with [`Qunit`](http://qunitjs.com/) as testing framework
+
+```yaml
+dist: trusty
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+language: node_js
+node_js:
+  - "0.12"
+env:
+    - EMBER_VERSION=default
+    # - EMBER_VERSION=release
+    # - EMBER_VERSION=beta
+    # - EMBER_VERSION=canary
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env: EMBER_VERSION=release
+    # - env: EMBER_VERSION=beta
+    # - env: EMBER_VERSION=canary
+
+before_install:
+    - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
+    - export DISPLAY=:99.0
```

Comment:
same kind of comments on each specific setting?

For example:
 `# starting a GUI to run tests, per https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI` or `# upgrade npm to 2.0` ...

What do you think?